website/docs/language/resources/ephemeral/index.mdx

```diff
@@ -44,21 +44,43 @@ Use write-only arguments to securely pass temporary values to resources during a
 <CodeBlockConfig highlight="11">
```

Comment:
@bschaatsbergen First of all thank you so much for this example! I _super_ appreciate it! 

How would you feel about moving the expanded `aws_secretsmanager_secret_version` logic to the bottom of the article in a separate `## Examples` section? I'd like this introduction section pretty basic while explaining how write-only arguments work, and then further down this expanded example on real-life usage would be _amazing_.